typescript/src/bitcoin.ts

```diff
@@ -624,3 +624,20 @@ export function locktimeToNumber(locktimeLE: Buffer | string): number {
 export function createOutputScriptFromAddress(address: string): Hex {
   return Hex.from(Script.fromAddress(address).toRaw().toString("hex"))
 }
+
+/**
+ * Returns the Bitcoin address based on the script pub key placed on the output
+ * of a Bitcoin transaction.
+ * @param scriptPubKey Scirpt pub key placed on the output of a Bitcoin
+ *        transaction.
+ * @param network Bitcoin network.
+ * @returns The Bitcoin address.
+ */
+export function getAddressFromScriptPubKey(
```

Comment:
The `scriptPubKey` is a field specific to transaction outputs but basically, this function converts any script to a Bitcoin address. We can make this function more generic and explicitly the opposite of `createOutputScriptFromAddress`. That said, I propose naming it `createAddressFromOutputScript`.

Reply:
c6f94f6